### PR TITLE
Adding new translation

### PR DESCRIPTION
**Pull request by latoulicious**
Adding few new translation

### CODE REVIEW

File: src/data/skill-name.csv [hunk 1]
```diff
@@ -303,7 +303,7 @@
 "逃げ直線○","Runner's Straight ○"
 "逃げコーナー◎","Runner's Corner ◎"
 "逃げコーナー○","Runner's Corner ○"
-"シックスセンス",""
+"シックスセンス","Six Sense"
```
Comment (stardustichi) [inline src/data/skill-name.csv:306]: this should be "Sixth Sense"

File: src/data/support-title.csv [hunk 2]
```diff
@@ -62,7 +62,7 @@
 "[押して忍べど燃ゆるもの]","[Unrestrainably Burning With Passion]"
 "[夢はホントに叶うんだ！]","[Dreams Do Come True!]"
 "[今ぞ盛りのさくら花]","[Sakura in Full Bloom]"
-"[花嫁たるもの！！]",""
+"[花嫁たるもの！！]","[I'm a bride!]"
```
Comment (stardustichi) [inline src/data/support-title.csv:65]: its more like [What Makes a Bride!!]

File: src/data/support-title.csv [hunk 3]
```diff
@@ -62,7 +62,7 @@
...
 "[召しませふぁーすとBite！]",""
-"[おもい、ねがう]",""
+"[おもい、ねがう]","[I Think and I Hope]"
```
Comment (stardustichi) [inline src/data/support-title.csv:67]: omoi here should be thought, or feeling. while negau kinda like wish. [A Feeling, a Wish]

File: src/data/trainer-title.csv [hunk 4]
```diff
@@ -196,7 +196,7 @@
 "ナリタブライアン名手",""
 "キミの夢へと走り出せ！","Chase Your Dreams"
 "Brand-new Friend","Brand-new Friend"
-"花咲く乙女のJunePride",""
+"花咲く乙女のJunePride","June Pride, A Flowering Maiden"
```
Comment (stardustichi) [inline src/data/trainer-title.csv:199]: "JunePride of the Blooming Maidens" or you can keep it as flowering, its kinda the same

**Review by stardustichi — CHANGES_REQUESTED** [review]
ive translated some but cant merge due to conflict, still needs review though

File: src/data/skill-name.csv (hunk 1 above)
Comment (latoulicious) [reply src/data/skill-name.csv]: Ah yes , i forgot how to write that

File: src/data/trainer-title.csv (hunk 4 above)
Comment (latoulicious) [reply src/data/trainer-title.csv]: Alright

File: src/data/support-title.csv (hunk 3 above)
Comment (latoulicious) [reply src/data/support-title.csv]: Yeah, i was confused with the translated too

File: src/data/support-title.csv (hunk 2 above)
Comment (latoulicious) [reply src/data/support-title.csv]: Alright

**Review by latoulicious — COMMENTED** [review]
Fixing the translation with help from stadustichi

**Review by FabulousCupcake — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
@stardustichi requested changes has been made; could you approve so we can merge this?

**Review by stardustichi — APPROVED** [review]

**Pull request merged**